libpod/options.go

```diff
@@ -2341,3 +2341,16 @@ func WithMountAllDevices() CtrCreateOption {
 		return nil
 	}
 }
+
+// WithNested sets the entry to write to the /etc/passwd file.
+func WithNested(nested bool) CtrCreateOption {
```

Comment:
Also here, `WithSELinuxNested`? Also please fix the comment to correctly describe what it does.